src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -848,12 +844,6 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 *  Magento now displays the customer middle name in the customer details on orders and in the new order email sent to customers. *Fix submitted by Sergiy Vasiutynskyi in pull request [24746](https://github.com/magento/magento2/pull/24746)*. [GitHub-23627](https://github.com/magento/magento2/issues/23627)
 
-### Payment methods
-
-<!--- ENGCOM-6124-->
-
-*  Magento now correctly stores the attribute `url_path` for non-default stores. *Fix submitted by Dmytro Androshchuk in pull request [25143](https://github.com/magento/magento2/pull/25143)*. [GitHub-25120](https://github.com/magento/magento2/issues/25120)
-
```

Comment:
Moved this part from the `Payment methods` section to the `URL rewrites` section, which makes a lot more sense.

Reply:
Thansk, Pieter -- I had already moved this is the .md files. I use the JIRA affected components for my initial categorization but always need to tweak that before GA. Thanks!